about.html

```diff
@@ -12,62 +12,79 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+  <header>
```

Comment:
Be careful here of getting caught in what I cal divception. You can take a more efficient approach to marking up the header as follows

```html
    <header>

        <nav> 
            <img src="img/lambda-black.png">
            <a href="index.html">Home</a>
            <a href="about.html">About</a>
            <a href="#">Products</a>
            <a href="#">Blog</a>
            <a href="#">Contact</a>
        </nav>
        
        <div class="top_img"> <!-- Centered header image -->
            <img src="img/jumbo.jpg">
        </div>

    </header>
```